dashboard/src/actions/repos.ts

```diff
@@ -128,72 +127,24 @@ export const installRepo = (
   repoURL: string,
   authHeader: string,
   customCA: string,
-  syncJobPodTemplate: string,
+  syncJobPodTemplate: any,
```

Comment:
mm, why isn't this a string now? it seems that we are still parsing it as a yaml below

Reply:
Whoops, yes - I changed the type on the incorrect function, I'd meant to verify that `shared/apprepository:Create` accepted an object. Thanks.